composer.json

```diff
@@ -18,11 +18,11 @@
         "php": "^7.4|^8.0",
         "dniccum/phone-number": "^1.1",
         "inspheric/nova-email-field": "^1.42",
-        "tipoff/authorization": "^2.3.1",
-        "tipoff/support": "^1.5.7"
+        "tipoff/authorization": "^2.3.0",
+        "tipoff/support": "^1.5.6"
     },
     "require-dev": {
-        "tipoff/test-support": "^1.2.0"
+        "tipoff/test-support": "^1.1.2"
```

Comment:
There was something wrong with Pacakgist not recognizing the new release tags. It appears to be resolved now.